scripts/e2e_test/e2e_test.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+require 'tmpdir'
+require 'json'
+
+class TestRunner
+  NUM_RUNS_TO_KEEP = 5
+  PROJECT_ROOT = File.expand_path(__dir__ + '/../..') 
+  RUNS_DIR = File.expand_path(__dir__ + '/runs')
+  CYPRESS_DIR = File.join(PROJECT_ROOT, 'packages/cli/cypress')
+
+  E2E_CONFIG = [
+    {
+      name: 'redwood_ts',
+      command: "yarn create redwood-app . --typescript; touch yarn.lock; yarn",
```

Comment:
first command fails because it notices yarn.lock in the root mailing directory.  this is gross but i think it works?  will add a comment about it if we keep it